static/js/stats/stats.js

```diff
@@ -82,12 +82,96 @@ $(document).ready(function () {
     });
 });
 
-function populate_messages_sent_over_time(data) {
-    if (data.end_times.length === 0) {
-        // TODO: do something nicer here
-        return;
-    }
+function replace_data_with_fixture_data() {
+    var trace1 = {
+      x: [1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010],
+      y: [10, 15, 13, 17, 10, 15, 13, 17, 10, 15, 13, 17],
+      type: 'bar',
+      visible: true,
+      name:'Humans',
+      hoverinfo: 'none',
+    };
+    var layout1 = {
+        barmode: 'group',
+        width: 750,
+        height: 400,
+        margin: { l: 40, r: 0, b: 40, t: 0 },
+        xaxis: {
+            fixedrange: true,
+            rangeslider: { bordercolor: '#D8D8D8', borderwidth: 1 },
+            type: 'date',
+        },
+        yaxis: { fixedrange: true, rangemode: 'tozero' },
+        legend: {
+            x: 0.75, y: 1.12, orientation: 'h', font: font_14pt,
+        },
+        font: font_14pt,
+    };
+    var layout2 = {
+        width: 750,
+        height: 370,
+        margin: {
+            l: 40, r: 0, b: 100, t: 20,
+        },
+        xaxis: {
+            fixedrange: true,
+            rangeselector: {
+                x: 0.808,
+                y: -0.2,
+                buttons: [
+                    {
+                        count: 30,
+                        label: 'Last 30 Days',
+                        step: 'day',
+                        stepmode: 'backward',
+                    },
+                    {
+                        step: 'all',
+                        label: 'All time',
+                    },
+                ],
+            },
+        },
+        yaxis: {
+            fixedrange: true,
+            rangemode: 'tozero',
+        },
+        font: font_14pt,
+    };
+    var trace3 = {
+        x: [1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010],
+        y: [16, 5, 11, 9, 1, 8, 3, 17, 10, 5, 13, 7],
+        type: 'scatter',
+        name: "Active users",
+        hoverinfo: 'none',
+        visible: true,
+    };
+    Plotly.plot('id_messages_sent_over_time', [trace1], layout1, {displayModeBar: false});
+    Plotly.newPlot('id_number_of_users', [trace3], layout2, {displayModeBar: false});
+    return;
+}
 
+function make_everything_opaque() {
+    document.getElementById("messages_timescale_anchor").style.opacity='0.2';
+    document.getElementById("graph_container").style.opacity='0.2';
+    document.getElementById("pie_messages_sent_by_client").style.opacity='0.2';
+    document.getElementById("pie_messages_sent_by_type").style.opacity='0.2';
+    document.getElementById("users_anchor").style.opacity='0.2';
+    document.getElementById("id_number_of_users").style.opacity='0.2';
+    document.getElementById("users_total").style.opacity='0.2';
+    return;
+}
+
+function show_insufficient_data_message() {
```

Comment:
For a one-liner like this, you can just put it in-line, instead of separating it out as a function.
Also, feel free to use jquery constructs.